src/main/java/duke/TripleConsumer.java

```diff
@@ -0,0 +1,12 @@
+package duke;
+
+public interface TripleConsumer {
+    /**
+     * Interface for our command.
+     *
+     * @param tasks
+     * @param ui
+     * @param storage
+     */
+    void execute(TaskList tasks, Ui ui, Storage storage);
+}
```

Comment:
I like how you package the three params into TripleConsumer and apply it by using Stream. Learned from you :) Thanks.